File: apresentacao/sinqia/src/app/movimento-manual/movimento-manual.ts
import { ProdutoCosif } from './../produto/produto-cosif/produto-cosif';
import { Produto } from './../produto/produto';
export interface MovimentoManual {
  Dat_Mes: number;
  Dat_Ano: number;
  Num_Lancamento: number;
  Cod_Produto: string;
  Cod_Cosif: string;
  Des_Descricao: string;
  Dat_Movimento: string;
  Cod_Usuario: string;
  Val_Valor: number;
  Produto : Produto;
  ProdutoCosif : ProdutoCosif;
}

File: backend-aplicacao/Sinqia/Sinqia.Domain/Entites/MovimentoManual.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Sinqia.Domain.Entites
{
    [Table("Movimento_Manual")]
    public class MovimentoManual
    {
       
        [Key]
        [Column(Order = 0)]
        public decimal Dat_Mes { get; set; }
        [Key]
        [Column(Order = 1)]
        public decimal Dat_Ano { get; set; }
        [Key]
        [Column(Order = 2)]
        public decimal Num_Lancamento { get; set; }
        [Key]
        [Column(Order = 3)]
        public string Cod_Produto { get; set; }
        [Key]
        [Column(Order = 4)]
        public string Cod_Cosif { get; set; }
        public string Des_Descricao { get; set; }
        public DateTime Dat_Movimento { get; set; }
        public string Cod_Usuario { get; set; } 
        public decimal Val_Valor  { get; set; }

        public virtual ProdutoCosif ProdutoCosif { get; set; }
        public virtual Produto Produto { get; set; }
    }
}
